master/reference/calicoctl/resources/policy.md

```diff
@@ -65,6 +65,52 @@ the policy is enforced after connection tracking and any DNAT.
 See [Using Calico to Secure Host Interfaces]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal)
 for how `doNotTrack` and `preDNAT` can be useful for host endpoints.
 
+The `types` field is useful when a policy specifies no ingress rules and you
+want that to mean that no ingress should be allowed, to each endpoint to which
+that policy applies, except as allowed by any other policies that apply to that
+endpoint.  You can express that by including `ingress` as one of the values of
+the `types` field - effectively saying "please apply this policy to ingress
+traffic, even though it looks like it should not."
+
+Similarly, when a policy specifies no egress rules and you want that to mean
+that no egress should be allowed ..., you can express that by including
+`egress` as one of the values of the `types` field.
+
```

Comment:
The above two paragraphs are oriented around the corner case scenario.  While this is the reason we have the `type` field, I think it would more consumable to orient the description around the straightforward cases.  i.e. Make it clear what you get when the user has specifies a value for the `type` field.  Then you can follow up with the more complex "what happens when you don't specify a `type` field" defaulting below. 

Another way of thinking about this is the behavior is obvious if people supply a value for the `type` field. We needed the defaulting of the `type` field for backwards compatibility, which is the corner case and an ever diminishing subset of users (given future users > current users).

Reply:
I agree with @lxpollitt that the paragraph should clearly describe what the types field does rather than dive directly into what it means in it's absence. The first sentence is difficult to follow because it tries to layout all the exceptions. Something to the effect: 
> The `types` field specifies whether the policy applies to ingress and/or egress traffic. 

And then get into what happens when it's missing.

Reply:
The first sentence is a difficult construction. Unfortunately, it's just not quite clear what you are trying to say. One of the issues is that the sentence is a bit long. Could you perhaps make this into two sentences? That might help a lot. It could also be that you are trying to accomplish too much with this single sentence as others have implied.

A nit as well, should be an em dash and not a hyphen in the second sentence, with no spaces around it. You can do this with either `&mdash;` or `—`.